docs/platform_concepts/engagement/cdp/user_data_segments/segment_usecases.md

```diff
@@ -58,6 +58,30 @@ To implement the use case, follow these steps:
 <img src="https://i.imgur.com/mUTWcrH.png"/>
 
 
+### Segment based on user created date
+
+Creating segments based on the created date can be useful in various use cases. You can use the user property `createdAt` to define your condition.
+
+Here are a few real-time examples of user segment based on the created date:
+
+* Onboarding Campaign: You can create a segment of customers who were recently created or onboarded within a specific time frame.
+* Re-engagement Campaign: Identify customers who haven't engaged with your business for a certain period since their creation. 
+* VIP Customers: Segment customers based on the date they became your customers and their purchase history.
+* Churn Prevention: Monitor customers who are nearing their anniversary or renewal date. 
+* Feedback Surveys: Create segments of customers who recently became your customers to gather feedback on their initial experience.
```

Comment:
A small suggestion: Create a segment of newly acquired customers to gather feedback on their initial experience with your business.